### PR TITLE
webui: Fix infinite loop when trying to log in with invalid account

### PR DESCRIPTION
**Pull request by fbergkemper**
Close socket on fwrite() error. fwrite() returns the number of bytes
written, or false on error.

Fixes #1324: Infinite loop when trying to log with invalid account

#### Please check

- [x] Short description and the purpose of this PR is present _above this paragraph_
- [x] Your name is present in the AUTHORS file (optional)

If you have any questions or problems, please give a comment in the PR.

### Helpful documentation and best practices

- [Git Workflow](https://docs.bareos.org/DeveloperGuide/gitworkflow.html)
- [Automatic Sourcecode Formatting](https://docs.bareos.org/DeveloperGuide/generaldevel.html#automatic-sourcecode-formatting)
- [Check your commit messages](https://docs.bareos.org/DeveloperGuide/gitworkflow.html#commits)

##### General

- [x] PR name is meaningful
- [x] Purpose of the PR is understood
- [x] Separate commit for this PR in the CHANGELOG.md, PR number referenced is same
- [x] Commit descriptions are understandable and well formatted

##### Source code quality

- [x] Source code changes are understandable
- [x] Variable and function names are meaningful
- [x] Code comments are correct (logically and spelling)
- [x] Required documentation changes are present and part of the PR
- [x] `bareos-check-sources --since-merge` does not report any problems
- [x] `git status` should not report modifications in the source tree after building and testing

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ and since Bareos version 20 this project adheres to [Semantic Versioning](https:
 - docs: declare shell scripts code blocks as "sh" instead of "shell-session" [PR #802]
 - [Issue #1205]: PHP 7.3 issue with compact() in HeadLink.php [PR #829]
 - reorder acquire on migrate/copy to avoid possible deadlock [PR #828]
+- [Issue #1324]: Infinite loop when trying to log with invalid account [PR #840]
```
Comment (pstorz) [inline CHANGELOG.md:34]:
```suggestion
- [Issue #1324]: Infinite loop when trying to log in with invalid account [PR #840]
```

**Review by pstorz — CHANGES_REQUESTED** [review]
Looks good, but it makes sense to change the title of the bug, and the commit message to have the correct sentence.

**Comment by fbergkemper** [comment]
> 
> 
> Looks good, but it makes sense to change the title of the bug, and the commit message to have the correct sentence.

Commit message reworded.

File: CHANGELOG.md (hunk 1 above)
Comment (fbergkemper) [reply CHANGELOG.md]: Requested change applied.

**Review by pstorz — APPROVED** [review]
Looks good to me. Good Work

**Pull request merged**